android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/MeasureReportViewModel.kt

```diff
@@ -266,9 +282,7 @@ constructor(
           end = endDateFormatted,
           reportType = POPULATION,
           subject = null,
-          practitioner = null
-          /* TODO DO NOT pass this id to MeasureProcessor as this is treated as subject if subject is null.
-          practitionerId?.asReference(ResourceType.Practitioner)?.reference*/ ,
+          practitioner = practitionerId?.asReference(ResourceType.Practitioner)?.reference,
```

Comment:
There is a `TODO` comment above this line instructing not to provide a non-null value as `practitionerId`. There is a possibility with this new change to have a non-nullable `practitionerId`. Was the bug resolved in `MeasureProcessor`?